app/src/lib/stats/stats-store.ts

```diff
@@ -1098,14 +1104,6 @@ export class StatsStore implements IStatsStore {
       m => ({ [k]: m[k] + n } as Pick<IDailyMeasures, keyof NumericMeasures>)
     )
 
-  /** Post some data to our stats endpoint. */
-  private post = (body: object) =>
-    fetch(StatsEndpoint, {
-      method: 'POST',
-      headers: new Headers({ 'Content-Type': 'application/json' }),
-      body: JSON.stringify(body),
-    })
```

Comment:
Looks good, but out of curiosity -> Are the changes in this file required or a stylistic preference?

Reply:
They're required because we're using the jest jsdom test environment now which means that fetch() is available now so our tests would attempt to use it and fail.

Reply:
I see the mock post below. I see testing was probably the reason here

Reply:
:P - your comment hadn't loaded yet when I responded. :P